source/windows/bcrypt_aes.c

```diff
@@ -412,7 +393,11 @@ static int s_aes_cbc_encrypt(
 
     struct aws_byte_buf final_to_encrypt = s_fill_in_overflow(cipher, &to_encrypt);
     struct aws_byte_cursor final_cur = aws_byte_cursor_from_buf(&final_to_encrypt);
-    int ret_val = s_aes_default_encrypt(cipher, &final_cur, out);
+    int ret_val = AWS_OP_SUCCESS;
+    if (final_cur.len > 0) {
+        s_aes_default_encrypt(cipher, &final_cur, out);
```

Comment:
still not checking result here

Reply:
ugh, im dumb. stashed the fix to merge in auto merges and never poped it